controllers/storagecluster/provider_server_test.go

```diff
@@ -321,7 +321,6 @@ func createSetupForOcsProviderTest(t *testing.T, allowRemoteStorageConsumers boo
 	}
 
 	os.Setenv(providerAPIServerImage, "fake-image")
-	os.Setenv(onboardingValidationKeysGeneratorImage, "fake-image")
```

Comment:
Are you going to add unit-tests in a separate PR?

Reply:
I missed adding that, I will add it in the same PR itself

Reply:
I will add the unit-test in different PR